scripts/build-helpers/patch_wasmedge_wat_sock_accept.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+if [[ "z$1" == "z" ]]
```

Comment:
Can you add a description on why this is necessary?

Reply:
Addressed this via
```
if [[ ! -f "$1" ]]
then
    echo "Pass target file as argument. Could not find a file at '$1'"
    exit 1
fi
```